src/modules/keyboardmanager/README.md

```diff
@@ -0,0 +1,101 @@
+# Keyboard Manager 
+
+The Keyboard Manager (KBM) is a keyboard remapper that allows a user to
+redefine keys on their keyboard (ex. swapping the letter A and D) as
+well as shortcuts (<kbd>Ctrl</kbd>+<kbd>C</kbd> to <kbd>Win</kbd>+<kbd>C</kbd>. You can use these remappings as long as KBM is enabled and PowerToys is running in the background. Below is an example of using keys and shortcuts that were remapped:
```

Comment:
missing ). Also didnt know about this `<kbd>` tag, we could use it for the A and D in previous line as well.